sysdig/resource_sysdig_secure_cloud_auth_account_component.go

```diff
@@ -0,0 +1,390 @@
+package sysdig
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	v2 "github.com/draios/terraform-provider-sysdig/sysdig/internal/client/v2"
+	cloudauth "github.com/draios/terraform-provider-sysdig/sysdig/internal/client/v2/cloudauth/go"
+)
+
+func resourceSysdigSecureCloudauthAccountComponent() *schema.Resource {
+	timeout := 5 * time.Minute
+
+	return &schema.Resource{
+		CreateContext: resourceSysdigSecureCloudauthAccountComponentCreate,
+		UpdateContext: resourceSysdigSecureCloudauthAccountComponentUpdate,
+		ReadContext:   resourceSysdigSecureCloudauthAccountComponentRead,
+		DeleteContext: resourceSysdigSecureCloudauthAccountComponentDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(timeout),
+			Update: schema.DefaultTimeout(timeout),
+			Read:   schema.DefaultTimeout(timeout),
+			Delete: schema.DefaultTimeout(timeout),
+		},
+		Schema: map[string]*schema.Schema{
```

Comment:
this schema definition seems to be largely duplicative of the component schema embedded within the account. The only difference being the presence of the `account_id` field. Can these definitions be consolidated, and merely add the account_id when needed at this top level context?

Reply:
Discussed this offline on the reasoning and consolidated this in latest commit.